app/models/PaymentMode.scala

```diff
@@ -0,0 +1,18 @@
+package models
+
+sealed trait PaymentMode {
```

Comment:
It looks nice! I think we have a couple other enums in our codebase, do you mind if migrate them all at once in a separate PR?